cmd/fdsn-ws/fdsn_event.go

```diff
@@ -472,8 +473,11 @@ func fdsnEventV1Handler(r *http.Request, h http.Header, b *bytes.Buffer) *weft.R
 			if err != nil {
 				return weft.ServiceUnavailableError(err)
 			}
-			// We don't provide EventLocationName
-			s := fmt.Sprintf("%s|%s|%.3f|%.3f|%.1f|GNS|GNS|GNS|%s|%s|%.1f|GNS|\n", eventID, tm.Format("2006-01-02T15:04:05"), latitude, longitude, depth, eventID, magType, magnitude)
+			loc := ""
+			if l, err := wgs84.ClosestNZ(latitude, longitude); err == nil {
+				loc = l.Name
```

Comment:
I think Nat was wanting l.Description() here  l.Name will just be the name of the locality which would be confusing I think.

Reply:
I've updated the code to use Description().